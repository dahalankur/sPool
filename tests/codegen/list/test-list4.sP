# test List replace

list<string> mylist = ["COMP 15", "is", "very", "interesting"]

List_replace(mylist, 0, "COMP 107") # replace in the front
List_replace(mylist, 2, "super") # replace in the middle
List_replace(mylist, 3, "cool") # replace at the end

int i = 0

while (i < List_len(mylist)):
    println(List_at(mylist, i))
    i = i + 1;
