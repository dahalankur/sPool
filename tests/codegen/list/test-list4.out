COMP 107
is
super
cool
